http/swagger.yml

```diff
@@ -4711,6 +4713,396 @@ components:
           type: string
         dialect:
           $ref: "#/components/schemas/Dialect"
+    File:
```

Comment:
I think we want to namespace all of these definitions to be `fluxast` or something. According to the OpenAPI spec the `components` can be extended to contain our own namespace.

https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#componentsObject 

https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#specificationExtensions

So maybe we should put these into a components field `x-fluxast` instead of `schemas`. But before we do that we should check that the code generators will still generate the structures if we do. 

There is a doc on how do use the code generators, I am digging it up. Once I find it I'll link it here.

Reply:
@nathanielc after doing some experimenting I found that the package used to validate the swagger specification (https://github.com/getkin/kin-openapi) does not accept such extensions.

Reply:
Sounds good, lets leave it as it is for now then.